blackbox/docs/sql/subselects.txt

```diff
@@ -36,6 +36,21 @@ Multiple levels of sub selects are supported::
     +--------------+-------------+
     SELECT 5 rows in set (... sec)
 
+Joins in the sub selects are supported:
```

Comment:
imho there is no need to prove the joins and nested subselects here, so remove both paragraphs